airbyte-cdk/python/airbyte_cdk/base_python/integration.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, spec_string):
 class Integration(object):
 
     # can be overridden to change an input config
-    def configure(self, config: json, temp_dir: str) -> json:
+    def configure(self, config: object, temp_dir: str) -> object:
```

Comment:
would suggest using `Mapping[str, Any]` instead of `object` throughout this class

Reply:
Makes sense. I wasn't guessing at the types here.